core/src/main/java/org/elasticsearch/search/suggest/SuggestUtils.java

```diff
@@ -171,12 +173,12 @@ public static SuggestMode resolveSuggestMode(String suggestMode) {
         }
     }
 
-    public static Suggest.Suggestion.Sort resolveSort(String sortVal) {
+    public static TermSuggestionBuilder.SortBy resolveSort(String sortVal) {
```

Comment:
Right, I missed that. They just appeared in the diff for the renaming. So if they can go away, even better.